pals/Mulch.md

```diff
@@ -0,0 +1,69 @@
+
+# Mulch
+
+> "I've got a _brilliant_ idea!!!"
+>
+> Mulch has been reading up on dungeoneering, thinks it'd be a great way to get Seed Capital for her mutant magical plant nursery.
+
+#### Species, Class, Level:
+Goblin Wizzard, Level 1
+
+#### Languages known:
+Common, Elven
+
+#### Attributes
+* Str: 8  (-1)
+* Con: 13 (+1)
+* Dex: 14 (+2)
+* Int: 18 (+4)
+* Wis: 14 (+2)
+* Cha: 12 (+1)
+
+#### Defense scores
+* AC: 14 (cloth)
+* Fort: 11
+* Ref: 14
+* Will: 14
+
+#### Other scores
+* Initiative: +2
+* Speed: 5
+* Hit Points: 23 
+* Surge Value: 5
+* Surges: 7
+* Vision: Low light
+
+#### Powers
+* Fade Away (Encounter)
+* Orb of Imposition (Encounter)
+* Ghost sound (Cantrip, At-will)
+* Light (Cantrip, At-will)
+* Mage hand (Cantrip, At-will)
+* Prestidigitation (Cantrip, At-will)
+* Cloud of Daggers (Spell, At-will)
+* Thunderwave (Spell, At-will)
+* Icy Terrain (Spell, Encounter)
+* Flaming Sphere (Spell, Daily)
+* Sleep (Spell, Daily)
+* Make Whole (Ritual)
+* Silence (Ritual)
+* Animal Messenger (Ritual)
```

Comment:
FYI: Riteby also has Silence, and Jo's cleric has Make Whole. There's no harm in doubling up (there's not many rituals at level 1 😞 ) but you might be interested in Brew Potion, which was changed to level 1 in errata, or Wizard's Curtain or Alarm from the old dragon magazine.